cloudant.go

```diff
@@ -53,12 +53,12 @@ type Index struct {
 }
 
 // NewClient ...
-func NewClient(username string, password string) (*Client, error) {
-	auth := couchdb.BasicAuth(username, password)
+func NewClient(username string, apikey string, password string) (*Client, error) {
+	auth := couchdb.BasicAuth(apikey, password)
```

Comment:
Actually, a better way would be to keep the old api and add a new "NewClient" method.
That way it won't break current applications.
So, keep : NewClient(username string, password string) as it is,
and add : NewClientWithAPIKey(username string, apikey string, password string)

What do you think?